### DISCUSSION TITLE
show miner window Linux(Fedora)

### DISCUSSION THREAD

**Discussion by lukras** [RainbowMiner/RainbowMiner#1893] [Q&A]
what to install and in which config file to show the miner window in Linux (Fedora)

**Comment by RainbowMiner** [comment]
Miner windows cannot be shown in Linux - but you could run `./minerlog.sh` in a console window.
This script will show you all running miners-in-one. It calls a little tool `rbmlog`.

Also, all miner output will be dumped into log files in folder `./Logs`
